changelog/7365.feature.rst

```diff
@@ -0,0 +1 @@
+Instances of :class:`Node <_pytest.nodes.Node>` now expose a `qual_name` property
```

Comment:
The changelog should explain what this property is. Asking foremost for myself as a reviewer :grinning: